client-v2/src/components/FrontsEdit/FrontCollectionsOverview.tsx

```diff
@@ -26,13 +36,58 @@ const Container = styled(ContentContainer)`
   width: 380px;
 `;
 
+const CollapseAllButtonContainer = styled('div')`
+  margin-top: 10px;
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+  :hover {
+    ${ButtonCircularWithTransition} {
+      background-color: ${({ theme }) =>
+        theme.shared.button.backgroundColorFocused};
+    }
+  }
+`;
+
+const CollapseAllDiv = styled('div')`
+  cursor: pointer;
+  font-family: TS3TextSans-Bold;
+  font-size: 14px;
+`;
+
+const CollapseAllLabel = styled('div')`
+  display: inline-block;
+`;
+
+const CollapseAllButton = ({
+  front,
+  closeAllCollections
+}: CollectionOverviewCollapseAllButtonProps) => (
+  <CollapseAllDiv
+    onClick={e => {
+      e.preventDefault();
+      closeAllCollections(front.collections);
+    }}
+  >
+    <ButtonCircularCaret small={true} active={true} preActive={false} />{' '}
```

Comment:
Just a style note here you can omit `={true}` here for the true ones boolean props.